### PR TITLE
Added support for Language Culture Name

### PR DESCRIPTION
**Pull request by thorsten**
Patch for #293

### CODE REVIEW

**Comment by ocombe** [comment]
Thanks, could you also update the readme to add the description of this new method ?

**Comment by thorsten** [comment]
Yes, sure! Will update the documentation asap.

File: src/translate.service.ts [hunk 1]
```diff
@@ -361,6 +361,11 @@ export class TranslateService {
         this.translations[lang] = undefined;
     }
 
+    /**
+     * Returns the language code name from the browser, e.g. "de"
+     *
+     * @returns {any}
```
Comment (ocombe) [inline src/translate.service.ts]: change `@returns {any}` to `@returns string`

File: src/translate.service.ts [hunk 2]
```diff
@@ -379,4 +384,20 @@ export class TranslateService {
 
         return browserLang;
     }
+
+    /**
+     * Returns the culture language code name from the browser, e.g. "de-DE"
+     *
+     * @returns {any}
```
Comment (ocombe) [inline src/translate.service.ts]: change `@returns {any}` to `@returns string`

File: src/translate.service.ts (hunk 1 above)
Comment (thorsten) [inline src/translate.service.ts]: ack

File: src/translate.service.ts (hunk 2 above)
Comment (thorsten) [inline src/translate.service.ts]: ack

**Comment by ocombe** [comment]
Perfect thanks

**Pull request merged**